### ISSUE TITLE
An error of poses_diff() function

### ISSUE THREAD

**Issue by muxizju** [fandulu/DD-Net#6]
```
def poses_diff(x):
    H, W = x.get_shape()[1], x.get_shape()[2]
    x = tf.subtract(x[:, :1, ...], x[:, :-1, ...])
    x = tf.image.resize_nearest_neighbor(x, size=[H.value, W.value], align_corners=False)  # should not alignment here
    return x
```
the subtract operation subtract **x[:, :1, ...]** and x[:, :-1, ...], while the first input might be wrong, it should be **x[:, 1:, ...]**. Current operation in fact subtract the first postion for all frames with array broadcast.

I have test that train the lite network on SHREC coarse data, the acc of validation is almost the same while the acc of training dataset is lower than validation. Which might be the reason of too much dropout. There might be still some space for improving.

If I'm wrong, please let me know

**Comment by pengfeiZhao1993** [comment]
yes, I also find this error, after fixing it , the accuracy gets better.

**Comment by fandulu** [comment]
Thanks very much for pointing out this issue. I had corrected them in the current version. As both of you had mentioned, such a correction did an improvement in SHREC data (for coarse data: It was improved from 94.6 to 94.8 with 1.82M parameters and from 91.8 to 94.1 with 0.15M parameters).

**Comment by muxizju** [comment]
> Thanks very much for pointing out this issue. I had corrected them in the current version. As both of you had mentioned, such a correction did an improvement in SHREC data (for coarse data: It was improved from 94.6 to 94.8 with 1.82M parameters and from 91.8 to 94.1 with 0.15M parameters).

great job! very fast and high accuracy. And have you ever tried this model on other dataset ,especially the NTU RGB+D dataset. which is very hot for skeleton based action recognition. and how is the performance

**Comment by fandulu** [comment]
Sorry, I may not try it since the skeleton contains much noise and loses context information, it is not as good as only using RGB in body action recognition and I stop further investigate it. This work was originally designed for hand gesture as we could obtain a more accurate 3D hand pose and it may not be related to context information. If you are working on NTU, stacking more modules (e.g., 2s-AGCN) might help to boost the performance of NTU up to ~96+(CV) and ~90+(CS).

**Comment by PrinceP** [comment]
@pengfeiZhao1993 @muxizju Can you please explain how to run the SHREC dataset.

**Comment by PrinceP** [comment]
Got it! Thanks
